src/Repositories/ExtensionRepository.php

```diff
@@ -136,8 +176,6 @@ public function createExtension(array $apiPackage)
 
         $this->refreshInstalledExtension($extension);
 
-        $this->flushCacheKey('flagrow.io.search.list');
```

Comment:
It was introduced here https://github.com/flagrow/bazaar/commit/c0a2b68acafddeb95795776c2b4508570a12f915